iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/Catalogs.java

```diff
@@ -222,6 +222,19 @@ public static Table registerTable(Configuration conf, Properties props, String m
     return new HadoopTables(conf).create(schema, spec, map, location);
   }
 
+  public static void renameTable(Configuration conf, Properties props, TableIdentifier to) {
+    String catalogName = props.getProperty(InputFormatConfig.CATALOG_NAME);
+
+    Optional<Catalog> catalog = loadCatalog(conf, catalogName);
+    if (catalog.isPresent()) {
+      String name = props.getProperty(NAME);
+      Preconditions.checkNotNull(name, "Table identifier not set");
+      catalog.get().renameTable(TableIdentifier.parse(name), to);
+    } else {
+      throw new RuntimeException("Rename from " + props.getProperty(NAME) + " to " + to + "failed");
```

Comment:
"failed" a white space is missing from the beginning of failed. Also I think the " has been failed" is more precise

Reply:
Also could be "Table rename from _Cars_ to _Cats_ has been failed". I would put the Rename Table into the exception message. It would be much cleaner for the reader which operation failed. I know it it is not a big deal just bit cleaner imho